client/components/blogging-prompt-card/style.scss

```diff
@@ -115,6 +115,28 @@
 			gap: 6px;
 		}
 	}
+	/* Responsive styles */
+	@include breakpoint-deprecated( "<660px" ) {
+		.blogging-prompt__prompt-container {
+			.blogging-prompt__prompt-text {
+				text-align: center;
+			}
+			.blogging-prompt__prompt-navigation {
+				gap: 10px;
+			}
+			button.navigation-link {
+				&:first-child {
+					margin-left: -36px;
+				}
+				&:last-child {
+					margin-right: -36px;
+				}
+			}
```

Comment:
These styles slightly push the buttons off-screen on `/home` :)

**Before:**
<img width="399" alt="Screenshot 2024-05-02 at 07 50 50" src="https://github.com/Automattic/wp-calypso/assets/43215253/95e41996-b907-4e85-938a-7372113409aa">

**After:**

<img width="372" alt="Screenshot 2024-05-02 at 07 49 51" src="https://github.com/Automattic/wp-calypso/assets/43215253/02b8d1a4-5ef9-4388-bc27-83d5870adc3a">

Reply:
Ah! Good catch! Thanks for that. I'll get that patched up. 🤗

Reply:
I think this should be fixed now.